libs/extensions/std/lang/src/http-extractor-meta-inf.ts

```diff
@@ -147,6 +147,19 @@ export class HttpExtractorMetaInformation extends BlockMetaInformation {
             }
           },
         },
+        followRedirects: {
+          type: PrimitiveValuetypes.Boolean,
+          defaultValue: true,
+          docs: {
+            description: 'Indicates, whether to follow redirects on get requests. If `false`, Redirects are disabled. Default `true`',
```

Comment:
Redirects are not disabled, just not followed ;).

```suggestion
            description: 'Indicates, whether to follow redirects on get requests. If `false`, redirects are not followed. Default `true`',
```

Reply:
Changed as suggested